integrated_channels/integrated_channel/admin/__init__.py

```diff
@@ -85,3 +89,13 @@ class ApiResponseRecordAdmin(admin.ModelAdmin):
     )
 
     list_per_page = 1000
+
+
+@admin.register(IntegratedChannelAPIRequestLogs)
+class CornerstoneAPIRequestLogAdmin(admin.ModelAdmin):
+    """
+    Django admin model for IntegratedChannelAPIRequestLogs.
+    """
+
+    class Meta:
+        model = IntegratedChannelAPIRequestLogs
```

Comment:
Shouldn't this class be removed now that we only have one table for all channels?

Reply:
My bad! 😮‍💨 removed